starters/express-apollo-prisma/README.md

```diff
@@ -0,0 +1,190 @@
+- `npm run docker:build` - Builds, (re)creates, starts, and attaches to containers to a service in the background.
+- `npm run docker:start` - Runs an existing docker container.
+- `npm run docker:stop` - Stops an existing docker container.
+- `npm run docker:down` - Stop and removes the docker container & network.
+- `npm run docker:remove` - Removes stopped docker service containers.
+
+### Database and Redis 
+To start up your API in dev mode with an active database connection, please follow the following steps:
+
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run docker:mount`
+3. run `npm run dev`
+
+The above steps will make sure your API connects to the database and Redis instances that get started up with docker. When you finish work, run `npm run infrastructure:stop` to stop your database and Redis containers.
+
+### Seeding
+<!-- TODO -->
+
+### Reset infrastructure
+
+To seed the database, you need to do the following steps:
+
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run docker:mount`
+<!-- TODO 3. add seeding -->
+
+### Production build
+
+The `npm run build` command compiles the typescript code into the `/dist` folder and generates a `package.json` file. To use it in production, for example in a docker container, one would copy the contents of the `/dist` folder, and then run `npm install` to have all the dependencies.
+
+### CORS Cross-Origin Resource Sharing
+
+The [Cross-Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) standard works by adding new HTTP headers that let servers describe which origins are permitted to read that information from a web browser. For security reasons, browsers restrict cross-origin HTTP requests initiated from scripts. This means that you cannot request data from web applications on 'https://domain-a.com' from 'https://domain-b.com/data.json'.
+
+This application accepts CORS from all origins by default. Some web applications may require you to add the HTTP header `'Access-Control-Allow-Origin': '*'` to allow access.
+
+In order to restrict origin URLs that can access your API, you need to add a list of comma-separated origin URLs in the `CORS_ALLOWED_ORIGINS` variable located in your `.env` file. For example `CORS_ALLOWED_ORIGINS="https://starter.dev"`. In case you need to access the API in a development environment i.e. a Sveltekit application, you can add the local URL `http://127.0.0.1` to the `CORS_ALLOWED_ORIGINS` variable as `CORS_ALLOWED_ORIGINS=https://starter.dev,http://127.0.0.1`.
+
+## Project Structure
+
+### Folder structure
+
+```text
+- prisma
+- src
+	- graphql
+    - data-sources
+    - mappers
+    - schema
+      - hello
+      - technology
+    - server-context
+    - utils
+  - main.ts
+```
+
+### MVC Pattern
```

Comment:
I'd love to have a huddle to get MVC explained to me. Not necessarily feedback for the PR, I just feel like I never fully grasped what MVC entails.